src/App.tsx

```diff
@@ -1,61 +1,121 @@
 import './App.scss';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const titleHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setTitleError(false);
+  };
+
+  const selectUserHandler = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+    setUserError(false);
+  };
+
+  const resetFieldsForm = () => {
+    setTitle('');
+    setUserId(0);
+  };
+
+  const handleOnSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    setTitleError(!title);
+    setUserError(!userId);
+
+    if (!title || !userId) {
```

Comment:
```suggestion
    if (!title.trim() || !userId.trim()) {
```
to avoid todo only with spaces
<img width="248" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/0c7a2d34-941c-45d4-8258-6b67d22a4580">